schedule/yam/agama_profile.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name: agama
+description: >
+  Perform interactive installation with agama.
+schedule:
+  - yam/agama/boot_agama
+  - yam/agama/import_agama_profile
+  - yam/agama/patch_agama_tests
+  - yam/agama/agama_bundle
+  - installation/grub_test
+  - installation/first_boot
```

Comment:
for booting existing module can be used for encryption.

Reply:
Updated